data/timeline.json.unvalidated.in

```diff
@@ -958,34 +957,778 @@
+            "data": {
+                "actor": "Ada",
+                "input": {
+                    "type": "choice",
+                    "settings": {
+                        "prompt": "Do you see a cool wave animation?",
+                        "choices": {
+                            "yes": {
+                                "text": "Yes"
+                            },
+                            "no": {
+                                "text": "No"
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "yes": [
+                        "chat::ada::processing::wave_change_num",
+                        "chat::ada::processing::wave_change_num_pause"
+                    ],
+                    "no": [
+                        "chat::ada::processing::wave_animation",
+                        "chat::ada::processing::wave_animation_play",
+                        "chat::ada::processing::wave_animation_play_pause"
+                    ]
+                }
+            }
+        },
+        {
+            "name": "chat::ada::processing::wave_change_num",
+            "type": "chat-actor",
+            "data": {
+                "actor": "Ada",
+                "message": "Now let's change the code and see what happens. In the Text Editor, let's change the value of the variable “num” on Line 1 from 20 to 10, then press Play."
+            }
+        },
+        {
+            "name": "chat::ada::processing::wave_change_num_pause",
+            "type": "input-user",
+            "data": {
+                "actor": "Ada",
+                "input": {
+                    "type": "choice",
+                    "settings": {
+                        "prompt": "Ready to continue?",
+                        "choices": {
+                            "yes": {
+                                "text": "Ready!"
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "yes": [
+                        "chat::ada::processing::wave_change_num_to_30",
+                        "chat::ada::processing::wave_change_step",
+                        "chat::ada::processing::wave_change_step_pause"
+                    ]
+                }
+            }
+        },
+        {
+            "name": "chat::ada::processing::wave_change_num_to_30",
+            "type": "chat-actor",
+            "data": {
+                "actor": "Ada",
+                "message": "Where did all our lines go? Let's change the num variable to 30."
+            }
+        },
+        {
+            "name": "chat::ada::processing::wave_change_step",
+            "type": "chat-actor",
+            "data": {
+                "actor": "Ada",
+                "message": "Next, let's change the space between the arcs. On Line 7, change the step command from 22 to 30, then press Play."
```

Comment:
since we above say `variable “num” on Line 1`, should we here do the same...
`change the  “step“ command from 22 to 30`

just a suggestion...